modules/core/src/main/java/org/apache/ignite/internal/IgniteComputeImpl.java

```diff
@@ -68,10 +68,11 @@ public class IgniteComputeImpl extends AsyncSupportAdapter<IgniteCompute>
     /** Custom executor name. */
     private String execName;
 
+    /** */
+    private boolean isSysTaskGuard;
```

Comment:
Could this be replaced with something explains it's a public/private api?

Reply:
Fixed. Changed to isPublicFacade.